samples/sensor/ina219/src/main.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021 Leonard Pollak
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <usb/usb_device.h>
+#include <drivers/sensor/ina219.h>
+
+void main(void)
+{
+#if defined(CONFIG_USB_UART_CONSOLE)
```

Comment:
Please remove USB device support, it is not necessary here.

Reply:
There is an overlay-usb.conf that can be used to enable USB device support.
Should I remove that one as well?

Reply:
yes, please.

Reply:
I have removed all USB related stuff from the sample code.